packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
These should all be dropped; minor metadata details in the example app scaffolding isn't relevant to package clients, so the changes are changelog exempt (under the documented "unless they only affect development of the package" exemption).

Reply:
gah! okay...